node/resolvers/search/index.ts

```diff
@@ -364,11 +365,7 @@ export const queries = {
       args
     )) as FacetsInput
 
-    let { fullText, searchState } = args
-
-    if (fullText) {
```

Comment:
Hey @chrsmutti 

We don't need this anymore, right?

If a Brazilian user searches for "maçã" on an American store, we don't need to translate it to "Apple" before sending the request to the back end. Am I right?

Reply:
Yes 👍